php-cs-fixer

```diff
@@ -29,6 +29,8 @@ if (defined('HHVM_VERSION_ID')) {
     if (getenv('PHP_CS_FIXER_IGNORE_ENV')) {
         fwrite(STDERR, "Ignoring environment requirements because `PHP_CS_FIXER_IGNORE_ENV` is set. Execution may be unstable.\n");
     } else {
+        fwrite(STDERR, "To ignore this requirement please set `PHP_CS_FIXER_IGNORE_ENV`.\n");
```

Comment:
I would add a note what can happen when ignoring. Sth like "if you use PHP version higher than supported, you may experience code modified in wrong way. Please report such cases via github"